Content.Shared/Roles/MindRoleComponent.cs

```diff
@@ -0,0 +1,53 @@
+using Content.Shared.Mind;
+using JetBrains.Annotations;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Roles;
+
+/// <summary>
+/// This holds data for, and indicates, a Mind Role entity
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+public sealed partial class MindRoleComponent : BaseMindRoleComponent
+{
+    /// <summary>
+    ///     Marks this Mind Role as Antagonist
+    ///     A single antag Mind Role is enough to make the owner mind count as Antagonist.
+    /// </summary>
+    [DataField]
+    public bool Antag { get; set; } = false;
+
+    /// <summary>
+    ///     True if this mindrole is an exclusive antagonist. Antag setting is not checked if this is True.
+    /// </summary>
+    [DataField]
+    public bool ExclusiveAntag { get; set; } = false;
+
+    /// <summary>
+    ///     The time this role was created
+    /// </summary>
+    public TimeSpan Created { get; set; } = TimeSpan.Zero;
```

Comment:
What is this for? It gets set but is never used?

Reply:
It was originally for the PR that this was making possible, and I guess I copied that field over by accident. But I no longer use that datafield on the future-change either, so I will just remove it